device.go

```diff
@@ -181,6 +190,19 @@ func virtioMmioBlkDeviceHandler(device pb.Device, spec *pb.Spec, s *sandbox) err
 	return updateSpecDeviceList(device, spec)
 }
 
+func virtioBlkCCWDeviceHandler(device pb.Device, spec *pb.Spec, s *sandbox) error {
```

Comment:
- You could change this to the following to make it clear the sandbox isn't used:
  ```go
  func virtioBlkCCWDeviceHandler(device pb.Device, spec *pb.Spec, _ *sandbox) error
  ```
- What if `spec == nil`?

Reply:
@jodh-intel thanks for the fast review ;)
So this is check in the function updateSpecDeviceList at https://github.com/kata-containers/agent/blob/5a109523cbf68a91a721c84f9e8762bd2cc6d34b/device.go#L228 and it is the same for all the handlers. See for example the [virtioBlkDeviceHandler](https://github.com/kata-containers/agent/blob/5a109523cbf68a91a721c84f9e8762bd2cc6d34b/device.go#L176)